### ISSUE TITLE
Plover recognizes two keys as same key

### ISSUE THREAD

**Issue by DemonfoxLux** [openstenoproject/plover#1681]
I've opened this issue on Yameroooo's repos: https://github.com/YAMEROOOO/plover-german-syllatype/issues/3

Basically, the basic english layout works perfectly fine.
After installing the german syllatype layout I made a small change to the layout itself since my keyboard has 10 instead of 11 columns, so I could fit all the keys onto  my keyboard.
Changed LAYOUT in system.py to: 
`LAYOUT = (
    "P- T- S- *- ^   -> -* -S -T -P "
    "K- F- N- R- ^   < -R -N -F -K "
             "A- E-  -I -U"
)`

However, when pressing virtual S1, it prints P, which is correct. But when I press virtual S2, which corresponds to K, it prints P as well.
As shown in the mentioned issue, the plugin translates the layout perfectly fine, S1 to P, S2 to K.

 - OS: Windows 11
 - Plover Version: Plover 4.0.0rc2+6.g53c416f

**Comment by DemonfoxLux** [comment]
I've been waiting for a comment, but also been on the lookout for a solution by scanning the source code.

The only problem I encountered so far: I can't build a working plover from the windows source code.
Problem appears to be when it comes to building wheels.
I've tried nearly everything the internet and logs suggested: installing setuptools, pyqt5, pyqt5-sip and wheel via pip, installing msvc 14, reinstalling python, just skipping the exception. Nothing helped.

Attached you will find the log file created by tox:
[1-install_deps.log](https://github.com/user-attachments/files/16488876/1-install_deps.log)


Additional information:
- OS: Windows 11
- Plover: Plover 4.0.0rc2+6.g53c416f source code (zip) from https://github.com/openstenoproject/plover/archive/refs/tags/continuous.zip
- Python: Python 3.12


Reason I did this is to further understand the source code and thus the background machinery.
Thanks in advance.

**Comment by DemonfoxLux** [comment]
Alright. After scanning through plover's source code I was able to determine the problem's cause to NOT be located in plover.
Furthermore, it's neither located in plover nor is it located in syllatype.

Instead it is my keyboard. I've made a custom keyboard based on https://github.com/mike-ady/Stenokey, and the chip sends S1 and S2 as the same code, resulting in said problem.
A more in-depth solution is found [here](https://github.com/mike-ady/Stenokey/issues/8)
